kvm/v13.0.0/README.md

```diff
@@ -0,0 +1,194 @@
+# :zap: Tenant Cluster Release v13.0.0 for KVM :zap:
+
+This release upgrades Kubernetes to 1.18 and Calico to 3.15. It also includes other minor component updates summarized below.
+
+## Change details
+
+
+### kubernetes [1.18.12](https://github.com/kubernetes/kubernetes/releases/tag/v1.18.12)
+
+Kubernetes 1.18 includes a large number of features, deprecations, and bug fixes. We have attempted to summarize the changes relevant to Giant Swarm customers in the following section. The full list of changes can be viewed [here](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.18.md).
+
+Please discuss with your Giant Swarm solution engineer if you are unsure about a cluster's Kubernetes 1.18 upgrade readiness.
+
+#### kube-apiserver
+- The following features are unconditionally enabled and the corresponding `--feature-gates` flags have been removed: `PodPriority`, `TaintNodesByCondition`, `ResourceQuotaScopeSelectors` and `ScheduleDaemonSetPods`.
```

Comment:
Is this too much detail? Tried to only mention things that could affect users.

Reply:
IMO this is good to know and not too much to include here